server/handlers/handlers.go

```diff
@@ -113,6 +117,8 @@ func (ah *ActivityHandler) Stop() {
 
 func (ah *ActivityHandler) Handle(activity msteams.Activity) error {
 	ah.queue <- activity
+	ah.plugin.GetMetrics().IncrementChangeEventQueueLength(activity.ChangeType)
```

Comment:
I queue after, and not before, since we might block if the queue is already full. But see my larger comment in the PR description for thoughts on this.